schema/swagger.yml

```diff
@@ -1723,6 +1777,27 @@ definitions:
         type: number
         example: 4368793
 
+  Forged:
```

Comment:
**Forged** is not any entity in our system like `Dapp`, `Block` or `Transaction`. So we should avoid using **Noun** term for it. Instead I prefer to name it `ForgingStats` or something more meaningful.